packages/plugin-react/src/index.ts

```diff
@@ -46,19 +52,39 @@ export interface Options {
 export default function viteReact(opts: Options = {}): PluginOption[] {
   // Provide default values for Rollup compat.
   let base = '/'
-  let filter = createFilter(opts.include, opts.exclude)
   let isProduction = true
   let projectRoot = process.cwd()
-  let skipFastRefresh = opts.fastRefresh === false
   let skipReactImport = false
+  let shouldProjectInclude: FileFilter
+  let shouldProjectExclude: FileFilter
+  let shouldUseFastRefresh: FileFilter
+  initializeFilters()
+
+  function initializeFilters(command?: string) {
+    shouldProjectInclude = createFileFilter(
+      { include: opts.babel?.include },
+      false,
+      projectRoot
+    )
+    shouldProjectExclude = createFileFilter(
+      { include: opts.babel?.exclude },
```

Comment:
Should this be "exclude" instead of "include"?

Reply:
No but it does need a comment to clarify that :)

It's creating a function that returns true if a `babel.exclude` pattern is matched.

Reply:
Ahh, got it, thanks!